maven-modules/maven-build-lifecycle/src/test/java/com/baeldung/mavenlifecycle/CourseAppUnitTest.java

```diff
@@ -0,0 +1,22 @@
+package com.baeldung.mavenlifecycle;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class CourseAppUnitTest {
+
+    @Mock
```

Comment:
Why are we using a mock here but not in the integration test?

Reply:
this is addressed